trouble_client/src/models/Player.js

```diff
@@ -0,0 +1,15 @@
+
+/** Represents a player in the game. */
+class Player{
```

Comment:
is this how we'll track if we are current player?

Reply:
Partially. This is just a class for representing a player on the front end.